app/models/consumer_role.rb

```diff
@@ -1157,12 +1157,12 @@ def update_all_verification_types(*args)
     end
   end
 
-  def admin_verification_action(admin_action, v_type, update_reason)
+  def admin_verification_action(admin_action, v_type, update_reason, admin_rejected)
     case admin_action
       when "verify"
         update_verification_type(v_type, update_reason)
       when "return_for_deficiency"
-        return_doc_for_deficiency(v_type, update_reason)
+        return_doc_for_deficiency(v_type, update_reason, admin_rejected)
```

Comment:
are we using this `return_for_deficiency` for anything else other than when admin rejects a document ?
As stated above, we can work without this `flag`